pyiron_atomistics/vasp/interactive.py

```diff
@@ -296,6 +296,16 @@ def _run_if_new(self, debug=False):
             self._check_incar_parameter(parameter="ISYM", value=0)
         super(VaspInteractive, self)._run_if_new(debug=debug)
 
+    def convergence_check(self):
+        """
+        We don't care about convergence for interactive jobs! Always returns True
+
+        Returns:
+            bool: Always True
+
+        """
+        return True
```

Comment:
```suggestion
        if self.server.run_mode.interactive:
            return True
```

We only want to skip that in interactive jobs, but sadly all our jobs always derive from `Interactive...`

Reply:
Unfortunately true

Reply:
Done!